BurnOutSharp/PackerType/WinZipSFX.cs

```diff
@@ -0,0 +1,494 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using System.Xml;
+using SharpCompress.Archives;
+using SharpCompress.Archives.Zip;
+
+namespace BurnOutSharp.PackerType
+{
+    public class WinZipSFX : IContentCheck, IScannable
+    {
+        /// <inheritdoc/>
+        public bool ShouldScan(byte[] magic) => true;
+
+        /// <inheritdoc/>
+        public string CheckContents(string file, byte[] fileContent, bool includePosition = false)
+        {
+            // Every single version has the string "WinZip Self-Extractor", and every 3+ version has a "version=" string due to internally including an XML
+
+            // "WinZip Self-Extractor"
+
+            byte[] check = new byte[] { 0x57, 0x69, 0x6E, 0x5A, 0x69, 0x70, 0x20, 0x53, 0x65, 0x6C, 0x66, 0x2D, 0x45, 0x78, 0x74, 0x72, 0x61, 0x63, 0x74, 0x6F, 0x72 };
+            if (fileContent.Contains(check, out int position))
+            {
+                // "<?xml"
+                byte[] check2 = new byte[] { 0x3C, 0x3F, 0x78, 0x6D, 0x6C };
+                if (fileContent.Contains(check2, out int xmlStartPosition))
+                    return $"WinZip SFX {GetV3PlusVersion(fileContent, position, xmlStartPosition)}" + (includePosition ? $" (Index {position})" : string.Empty);
+                else
+                    return $"WinZip SFX {GetV2Version(fileContent, position)}" + (includePosition ? $" (Index {position})" : string.Empty);
+            }
+
+            return null;
+        }
+
+        /// <inheritdoc/>
+        public Dictionary<string, List<string>> Scan(Scanner scanner, string file)
+        {
+            if (!File.Exists(file))
+                return null;
+
+            using (var fs = File.OpenRead(file))
+            {
+                return Scan(scanner, fs, file);
+            }
+        }
+        /// <inheritdoc/>
```

Comment:
Nitpick: Please add a space between the end of `Scan` and the beginning of the method comment.